### PR TITLE
Nearby holograms command

### PR DESCRIPTION
**Pull request by Matt-MX**
Adds the ability to run `/holo near <radius:integer>` (#128) to return a list of nearby holograms in a defined radius.

Not tested yet but should work. Very useful feature for administration and debugging.

(dont know how to get rid of pull upstream for my previous changes oops)

### CODE REVIEW

**Review by OakLoaf — APPROVED** [review]
This looks good to me

File: src/main/java/de/oliver/fancyholograms/commands/HologramCMD.java [hunk 1]
```diff
@@ -75,6 +75,9 @@ public boolean execute(@NotNull CommandSender sender, @NotNull String label, @No
             return new CreateCMD().run(sender, null, args);
         }
 
+        if (args[0].equalsIgnoreCase("near")) {
```
Comment (OliverSchlueter) [inline src/main/java/de/oliver/fancyholograms/commands/HologramCMD.java:78]: I would call the command "nearby" to match with FancyNpcs

File: src/main/java/de/oliver/fancyholograms/commands/HologramCMD.java [hunk 2]
```diff
@@ -118,7 +121,7 @@ public boolean execute(@NotNull CommandSender sender, @NotNull String label, @No
 
         // /holo {tab:action}
         if (args.length == 1) {
-            return Stream.of("help", "list", "teleport", "create", "remove", "edit", "copy", "info").filter(input -> input.startsWith(args[0].toLowerCase(Locale.ROOT))).toList();
+            return Stream.of("help", "list", "teleport", "create", "remove", "edit", "copy", "info", "near").filter(input -> input.startsWith(args[0].toLowerCase(Locale.ROOT))).toList();
```
Comment (OliverSchlueter) [inline src/main/java/de/oliver/fancyholograms/commands/HologramCMD.java:124]: rename to "nearby"

**Review by OliverSchlueter — COMMENTED** [review]
Have you tested the command?

**Comment by Matt-MX** [comment]
> Have you tested the command?

Yep it's working, tested in 1.20.4 and 1.21 👍

**Review by OliverSchlueter — APPROVED** [review]

**Comment by OliverSchlueter** [comment]
Thank you!

**Pull request merged**